programs/bpf_loader/src/serialization.rs

```diff
@@ -437,24 +437,26 @@ fn serialize_parameters_aligned(
                 size += size_of::<u8>() // is_signer
                 + size_of::<u8>() // is_writable
                 + size_of::<u8>() // executable
-                + size_of::<u32>() // original_data_len
+                + 4 // padding  of 4 bytes after executable
+                + size_of::<u64>() // original_data_len
```

Comment:
Why is this padding necessary?

Reply:
because of this : https://github.com/anza-xyz/agave/blob/f36a62dce8bbd9ee41c93626e86872241df31119/programs/bpf_loader/src/serialization.rs#L471

Reply:
That matched up with ` + size_of::<u32>() // original_data_len`